must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -205,14 +205,32 @@ for ns in $namespaces; do
 
     oc delete -f pod_helper.yaml
 
+    crash_collection(){
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
+            sleep 3
+            echo "waiting for the debug pod to be in ready state"
+        done
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
+    }
+
+    # creating a counter variable for collecting PID in array
+    count=1
```

Comment:
Rename this to something like `index` or `idx`. Any reason it starts at 1? Otherwise since it's an array index it should start at 0.

Reply:
it is for indexing. It should start with zero. I'll update this also.

Reply:
I don't mean to go too deep into this but you don't even need to declare the index. You can append in the array directly: `array+=($pid)`. And initialize the array with `array=()`.

Reply:
Also valid, but if it works don't break it. :joy: